app-ios/Modules/Sources/Timetable/TimetableListItemView.swift

```diff
@@ -5,17 +5,17 @@ import Theme
 
 struct TimetableListItemView: View {
     let timetableItemWithFavorite: TimetableItemWithFavorite
-
+    
     var timetableItem: TimetableItem {
         self.timetableItemWithFavorite.timetableItem
     }
-
+    
     var body: some View {
         HStack(alignment: .top) {
             VStack(alignment: .leading, spacing: 0) {
-                Spacer().frame(height: 4)
+                Spacer().frame(height: 16)
                 // TODO apply like flexbox layout
-                LazyHStack(spacing: 4) {
+                HStack(spacing: 4) {
```

Comment:
If LazyHStack is used, the title text is displayed but only 1 line, so changed it to HStack.